src/main/java/frc/robot/Constants.java

```diff
@@ -260,4 +260,12 @@ public static class ShooterConstants {
 
     public static final double kShootingTolerance = 0.05;
   }
-}
+  public static class ClimberConstants {
+    public static final int m_climberLeftMotorID = 20;
```

Comment:
Why ID 20 and 21? Aren't there lower unused IDs

Reply:
I'm honestly unsure